tests/ixia/test_ixia_traffic_restpy.py

```diff
@@ -0,0 +1,198 @@
+    vPort1.L1Config.NovusHundredGigLan.EnableAutoNegotiation =False
+    vPort1.L1Config.NovusHundredGigLan.EnableRsFec = True
+    vPort1.L1Config.NovusHundredGigLan.EnableRsFecStats = True
+
+    vPort2.L1Config.NovusHundredGigLan.IeeeL1Defaults  =  False
+    vPort2.L1Config.NovusHundredGigLan.EnableAutoNegotiation =False
+    vPort2.L1Config.NovusHundredGigLan.EnableRsFec = True
+    vPort2.L1Config.NovusHundredGigLan.EnableRsFecStats = True 
+
+    vPort3.L1Config.NovusHundredGigLan.IeeeL1Defaults  =  False
+    vPort3.L1Config.NovusHundredGigLan.EnableAutoNegotiation =False
+    vPort3.L1Config.NovusHundredGigLan.EnableRsFec = True
+    vPort3.L1Config.NovusHundredGigLan.EnableRsFecStats = True 
+
+    vPort4.L1Config.NovusHundredGigLan.IeeeL1Defaults  =  False
+    vPort4.L1Config.NovusHundredGigLan.EnableAutoNegotiation =False
+    vPort4.L1Config.NovusHundredGigLan.EnableRsFec = True
+    vPort4.L1Config.NovusHundredGigLan.EnableRsFecStats = True 
+
+    vPort5.L1Config.NovusHundredGigLan.IeeeL1Defaults  =  False
+    vPort5.L1Config.NovusHundredGigLan.EnableAutoNegotiation =False
+    vPort5.L1Config.NovusHundredGigLan.EnableRsFec = True
+    vPort5.L1Config.NovusHundredGigLan.EnableRsFecStats = True 
+
+    vPort6.L1Config.NovusHundredGigLan.IeeeL1Defaults  =  False
+    vPort6.L1Config.NovusHundredGigLan.EnableAutoNegotiation =False
+    vPort6.L1Config.NovusHundredGigLan.EnableRsFec = True
+    vPort6.L1Config.NovusHundredGigLan.EnableRsFecStats = True 
+
+    vPort1.Name = 'Tx1'
+    vPort4.Name = 'Rx1'
+    state = vPort1.State
+    print ('creating topology')
+
+    topology1    = ixNetwork.Topology.add(Name='Topo1', Ports=[vPort1, vPort2, vPort3, vPort4, vPort5, vPort6])
+    deviceGroup1 = topology1.DeviceGroup.add(Name='DG1', Multiplier='1')
+    ethernet1    = deviceGroup1.Ethernet.add(Name='Eth1')
+    ipv4         = ethernet1.Ipv4.add(Name='Ipv4')
+
+    ipv4.GatewayIp.ValueList(['192.168.1.1','192.168.1.1','192.168.1.1','192.168.1.1', '192.168.1.1', '192.168.1.1'])
+    ipv4.Address.ValueList(['192.168.1.2','192.168.1.3','192.168.1.4','192.168.1.5', '192.168.1.6', '192.168.1.7'])
+
+    ixNetwork.StartAllProtocols()
+    time.sleep(60) 
+
+    # Traffic
+    traffic_item = ixNetwork.Traffic.TrafficItem.add(Name='Traffic Test', TrafficType='ipv4')
+    dest = [{'arg1': '/api/v1/sessions/1/ixnetwork/topology/1/deviceGroup/1/ethernet/1/ipv4/1', 'arg2': 1, 'arg3': 4, 'arg4': 1 ,'arg5': 1}]
+    src  = [{'arg1': '/api/v1/sessions/1/ixnetwork/topology/1/deviceGroup/1/ethernet/1/ipv4/1', 'arg2': 1, 'arg3': 1, 'arg4': 1, 'arg5': 1}]
+    endPoint = traffic_item.EndpointSet.add()
+    traffic_item.Tracking.find().TrackBy = ['trackingenabled0']
+    endPoint.ScalableSources = src
+    endPoint.ScalableDestinations = dest
+    traffic_item.Generate()
+    ixNetwork.Traffic.Apply()
+    ixNetwork.Traffic.Start()
+    time.sleep(10)
+    print(session.StatViewAssistant('Traffic Item Statistics'))
```

Comment:
Please remove this debug code, or use logger to generate a log entry if  necessary.

Reply:
I will do that

Reply:
I have address this issue. print statements are replaced by logger.info()